src/state.js

```diff
@@ -7,6 +7,7 @@
 const State: {[state: string]: string} = {
   LOADING: 'loading',
   LOADED: 'loaded',
+  PROCESS: 'process',
```

Comment:
`PROCESS` is a misleading name as actually there is no any process between `AD_BREAK_START` and `AD_STARTED`. maybe `READY` is a better name for this state.

Reply:
@Yuvalke change it to PENDING.
The ad is loaded and intent to play was given but ad is not yet in PLAYING state.